### PR TITLE
Remove irrevalent tools

### PR DESCRIPTION
**Pull request by zhimin-z**
This PR aims to remove irrelevant tools for the corresponding categories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,7 +121,6 @@ This repository contains a curated list of awesome open source libraries that wi
 
 ## Model and Data Versioning
 * [Aim](https://github.com/aimhubio/aim) ![](https://img.shields.io/github/stars/aimhubio/aim?style=social) - A super-easy way to record, search and compare AI experiments.
-* [Apache Marvin](https://github.com/apache/incubator-marvin) ![](https://img.shields.io/github/stars/apache/incubator-marvin.svg?style=social) is a platform for model deployment and versioning that hides all complexity under the hood: data scientists just need to set up the server and write their code in an extended jupyter notebook.
```
Comment (axsaucedo) [inline README.md:124]: Why removing marvin? Seems may be slipping into not as widely maintained but it's not archived so given still backed by apache foundation would assume still active

**Review by axsaucedo — CHANGES_REQUESTED** [review]
Thanks for the contribution, I added a few questions to understand why projects are being removed as these all seem to align with CONTRIBUTING.md requirements, particularly the metadata managment systems

**Comment by axsaucedo** [comment]
@zhimin-z thanks for re-adding atlas - it seems the rest of the comments are not addressed, I'm not yet clear of the thinking to remove these tools, would you be able to provide more details?

**Comment by zhimin-z** [comment]
> @zhimin-z thanks for re-adding atlas - it seems the rest of the comments are not addressed, I'm not yet clear of the thinking to remove these tools, would you be able to provide more details?

I left the details in my replies. Which removal makes you confused? @axsaucedo

**Comment by zhimin-z** [comment]
> I am not sure if the comments were shared - perhaps you need to apply the review? I can't see any replies on the current comments

Hi, @axsaucedo Can you see my followup comments [here](https://github.com/EthicalML/awesome-production-machine-learning/pull/284/files/871fc2cfcce64cb97dc181cd656ae41891c90922)?

**Comment by axsaucedo** [comment]
No there are no comments
image

I assume the review hasn't been posted

**Comment by zhimin-z** [comment]
> No there are no comments image
> 
> I assume the review hasn't been posted

OK, that is weird. I would paste my comments here directly.

To begin with, allow me to clarify the logic behind the awesome list for production machine learning. A tool shall be listed here due to its nature that it _helps with the production of the machine learning lifecycle_, **AND** it should also either claim to help with the machine learning lifecycle (i.e. written explicitly in their documentation: "This tool aims to help with machine learning .......") or be widely recognized as a helper tool to machine learning lifecycle by the practitioners' community (i.e. de facto standards in ML production).

For example, tools like Ubuntu, Docker, NumPy, etc. can contribute to the machine learning lifecycle, and they are amazing tools, but we shall not list them here since they are not designed or specifically used for the machine learning process.

Considering this belief, I suggest removing 1) Metacat, 2) Marquez, and 3) Apache Marvi from our list.

1. Metacat
If you check its specification in its repo (it has no documentation), Metacat aims to provide a data discovery service. However, this does not necessarily correlate with machine learning. Out of curiosity, I also Googled this tool and attempted to discover how this tool can be used in a typical machine-learning lifecycle. I failed to locate any related material online. Maybe this tool can provide help in the training data discovery process, but I think it might not a good choice to keep it on the list for now. If someday the tools have better integration into the machine learning pipeline, which is yet widely recognized in the ML community or explicitly documented in its tool readme, then we should certainly contain it in our list.

2. Marquez
Similarly, Markquez seems more likely integrated into a typical ML pipeline than Metacat after I got a head-on experience with this tool. However, it still fails to either document how it can help a typical ML process **or** be recognized by the ML community widely. Several Google searches gave me no results on how it relates directly to machine learning production. 

3. Apache Marvi
According to its official documentation, there seems to be no way to figure out what this tool is used for. And I also check recent update content and doubt if this tool is literally dead or unmaintained. If we are unsure if this tool can help with model versioning or data versioning, then, we should not keep it on the model/data versioning module. Also, the star number of its repository is exactly 100, failing to meet our inclusion standard for over 100 stars. 100 stars for 4 years as an Apache tool indicates it indeed drew far less attention than its peers if any.

To sum up, our list aims to serve as the best tool catalog for the ML community in the foreseeable future. Then we should keep our bar for tool inclusion specifically for machine learning production. If the tools are not specifically used/designed for the machine learning process, then we shall add to our lie. What do you think? @axsaucedo

**Comment by axsaucedo** [comment]
I've seen git doing this with comments when they are staged (i.e. commented as "bulk review") - thansk for sharing them in text form.

I do like the explicit definition of: `It helps with the production of the machine learning lifecycle, AND it should also either claim to help with the machine learning lifecycle (i.e. written explicitly in their documentation: "This tool aims to help with machine learning .......") or be widely recognized as a helper tool to machine learning lifecycle by the practitioners' community (i.e. de facto standards in ML production).`

I am keen to add this to the CONTRIBUTING.md, so people make the case for tools when adding one.

In regards to the ones to be removed.

#### 1. metacat
I see what you mean, I actually gave a talk recently about [metadata and service discovery](https://www.youtube.com/watch?v=wzbj7LcQ2Ek) in MLOPs and how it's different to the DataOps world. I agree that these data discovery services are limited, and could certainly be used for ML training data. I would still be keen to keep some of these if they are actually used in some way in ML, particularly with training data. Looking at some of the key posts about metacat they do reference quite a lot the usage of [Jupyter notebooks](https://netflixtechblog.com/metacat-making-big-data-discoverable-and-meaningful-at-netflix-56fb36a53520) which would suggest that data science / data analyst practitioners would be actively using this. Given this is analogous to Atlas I would be more inclined to keep.

#### 2. Marquez 
For Marquez and generally ETL platforms its easier to argue their fit for MLOps, there are some usecases referencing marquez for MLOPS (e.g. https://www.slideshare.net/WillyLulciuc/marquez-an-open-source-metadata-service-for-ml-platforms) so I would be inclined to keep this one (also as it's part of the LF AI & Data Foundation, ensuring it has interoperability with such projects).

#### 3. Apache Marvin
Yes good points, let's remove.

**Comment by zhimin-z** [comment]
changed! @axsaucedo

**Review by axsaucedo — APPROVED** [review]
Great - thank you for the contribution
